Dockerfile

```diff
@@ -0,0 +1,19 @@
+FROM node:7.7.1-alpine
+
+RUN mkdir -p /usr/local/ui \
+&& apk update \
+&& apk upgrade \
+&& apk add -q vim \
+&& apk add -q nano;
+
+WORKDIR /usr/local/ui
+
+COPY package.json package.json
+COPY yarn.lock yarn.lock
+RUN yarn
+
+COPY . .
+RUN yarn run build
+
+EXPOSE 3000
+ENTRYPOINT ["npm", "start"]
```

Comment:
When we run the UI locally now, it does not use files in the build directory. I just want to make sure when we run the UI in the Docker container, it will run differently and really use the files in the build directory instead of files in the src directory.